lib/space.rb

```diff
@@ -1,4 +1,16 @@
 class Space
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'makers_bnb_test')
+    else
+      connection = PG.connect(dbname: 'makers_bnb')
+    end
+
+    result = connection.exec("SELECT * FROM spaces")
+    result.map { |space| space['name']}
```

Comment:
assume that eventually this will return instances of space class. We have made a 'name' attribute and instance (discuss further)